docs/infrastructure/faq.md

```diff
@@ -60,6 +60,22 @@ This error usually indicates that the Fleet server has run out of file descripto
 
 This could be caused by a mismatched connection limit between the Fleet server and the MySQL server that prevents Fleet from fully utilizing the database. First [determine how many open connections your MySQL server supports](https://dev.mysql.com/doc/refman/8.0/en/too-many-connections.html). Now set the [`--mysql_max_open_conns`](./configuring-the-fleet-binary.md#mysql_max_open_conns) and [`--mysql_max_idle_conns`](./configuring-the-fleet-binary.md#mysql_max_idle_conns) flags appropriately.
 
+## Why am I receiving a database connection error when attempting to "prepare" the databse?
+
+First, check if you have a version of MySQL installed that is at least 5.7. Then, make sure that you currently have a MySQL server running.
+
+The next step is to make sure the credentials for the database match what is expected. Test your ability to connect to the database with `mysql -u<username> -h<hostname_or_ip> -P<port> -D<database_name> -p`.
+
+If you're successful connecting to the database and still receive a database connection error, you may need to specify your database credentials when running `fleet prepare db`. It's encouraged to put your database credentials in environment variables or a config file.
+
+```
+$ /fleet prepare db \
```

Comment:
```suggestion
fleet prepare db \
```
@mikermcneil and I were discussing trying to leave out `$` from shell commands since this can make it more difficult to copy/paste them.